src/components/inputs/rich-text-input/editor.js

```diff
@@ -192,200 +40,27 @@ const Editor = props => {
         return (
           <Constraints.Horizontal constraint={props.horizontalConstraint}>
             <Spacings.Stack scale="xs">
-              <Container
+              <RichTextBody
+                ref={{
+                  containerRef: ref,
```

Comment:
passing multiple refs with useRef is pretty weird 🤔

Reply:
Maybe the RichTextInput could have its own context which creates the refs (among other things) so we can just ‚const { refs } = useRichTextInputContext()‘ of which one eg forwards ‚refs.container‘.

Reply:
One of the refs comes from `CollapsibleMotion` though. AFAIK I can't create a collapsibleMotion hook because of how emotion is built